packages/codemods/src/bin/spirit-codemod.js

```diff
@@ -0,0 +1 @@
+console.log('spirit-codemods');
```

Comment:
Please, add a comment with eslint disable and link this to another issue :-)

Reply:
comment added